### PR TITLE
07 25

### PR DESCRIPTION
**Pull request by LoganWright**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -14,30 +14,27 @@ let package = Package(
...
+        .Package(url: "https://github.com/qutheory/fluent.git", majorVersion: 0, minor: 0),
 
         //Allows complex key path subscripts
-        .Package(url: "https://github.com/qutheory/path-indexable.git", majorVersion: 0, minor: 2),
-        
+        .Package(url: "https://github.com/qutheory/path-indexable.git", majorVersion: 0, minor: 3),
+
         // Syntax for easily accessing values from generic data.
-        .Package(url: "https://github.com/qutheory/polymorphic.git", majorVersion: 0, minor: 2),
+        .Package(url: "https://github.com/qutheory/polymorphic.git", majorVersion: 0, minor: 3),
```
Comment (tannernelson) [inline Package.swift]: Should we just bring in Node?

Comment (tannernelson) [inline Package.swift]: Well, Fluent `@_export`s Node so we don't really need any of these.

File: Sources/Vapor/Fluent/Model.swift [hunk 2]
```diff
@@ -9,28 +12,30 @@ extension Model {
...
-            } else {
-                jsonValue = .null
-            }
-
-            json[key] = jsonValue
-        }
-
-        return JSON(json)
+    public func makeJSON() throws -> JSON {
+        fatalError("Swap for real json")
```
Comment (tannernelson) [inline Sources/Vapor/Fluent/Model.swift]: I can fix this in a different PR.

Comment (LoganWright) [inline Sources/Vapor/Fluent/Model.swift]: thank you @tannernelson I was going to ask how to fix this 👍

File: Package.swift (hunk 1 above)
Comment (LoganWright) [inline Package.swift]: I'll pull em out 👍

**Pull request merged**